src/main/scala/com/thirdship/libunit/CompoundTSUnit.scala

```diff
@@ -165,17 +165,22 @@ class CompoundTSUnit(val numerator: List[TSUnit] = List.empty[TSUnit],
 		}
 	}
 
+	private def keySetEquals(unit: CompoundTSUnit): Boolean = {
+		numeratorDimensions.keySet.equals(unit.numeratorDimensions.keySet) &&
```

Comment:
If I put it back into its original function, the cyclomatic complexity of the overall function will be exceeding the threshold set by ScalaStyle. What would you suggest: ignore the condition, find something else to extract, or find a better way of extracting the above function?